components/kubeflow/pytorch-launcher/src/launch_pytorchjob.py

```diff
@@ -116,33 +107,31 @@ def main(args):
         ),
         spec=jobSpec,
     )
+    logging.info('Creating TrainingClient.')
 
-    serialized_job = k8s_client.ApiClient().sanitize_for_serialization(job)
-
-    logging.info('Creating launcher client.')
-
+    # remove one of these depending on where you are running this
     config.load_incluster_config()
-    api_client = k8s_client.ApiClient()
-    launcher_client = launch_crd.K8sCR(
-        group=args.jobGroup,
-        plural=args.jobPlural,
-        version=args.version,
-        client=api_client
-    )
+    #config.load_kube_config()
```

Comment:
This might be needed?

Reply:
Here I have included both `config.load_incluster_config()` and `config.load_kube_config()` for use depending on how it will be run. I can update this to what ever makes most sense for users. This is what is included currently:
```
 # remove one of these depending on where you are running this
    config.load_incluster_config()
    #config.load_kube_config()
```

Reply:
I think the current approach makes sense, the pod is running in a container, on k8s, so it makes sense we'd leverage the pod's SA's creds here.